test/response_to_json_translator_test.cc

```diff
@@ -768,6 +781,85 @@ TEST_F(ResponseToJsonTranslatorTest, StreamingDirectTest) {
   EXPECT_TRUE(translator.Finished());
 }
 
+TEST_F(ResponseToJsonTranslatorTest, StreamingNewlineDelimitedDirectTest) {
```

Comment:
This test is almost the same as the original one. 

can we have a test to verify the json object delimiter is different?

Reply:
Using `ExpectJsonObjectEq` instead of `tester.TestElement` (which tests inputs with the expectation that they're formatted as a single JSON Array) verifies that every translated message is a valid JSON Object, with no extraneous `[]` wrappers or `,` delimiters that would cause `ExpectJsonObjectEq` to fail.

Let me see if I can come up with a more explicit test that each message ends with `'\n'` before conversions happen, though.

Reply:
@qiwzhang [here's an attempt at getting more specific with the newline-delimiter tests](https://github.com/grpc-ecosystem/grpc-httpjson-transcoding/pull/73/commits/7f59c322b007f699db724e810a3514601923004c). Let me know if that's sufficient.

Reply:
LGTM